### ISSUE TITLE
Model Size of Pruned model

### ISSUE THREAD

**Issue by AIbeginner2020** [VainF/Torch-Pruning#35]
I'm just starting on Model pruning and your work really helps a lot, I would really like to know how did you calculate the pruned Model size, thank you.

**Comment by Aarsh2001** [comment]
So you can either use torch_summary(package) if you want all the parameter details and model size which outputs something like this
<img width="597" alt="Screenshot 2021-05-22 at 11 35 07 AM" src="https://user-images.githubusercontent.com/65212523/119216539-dba76f00-baf1-11eb-8664-e204097d59b9.png">
 if you only want size you can do something like this 

```
def print_size_of_model(model):
    torch.save(model.state_dict(), "temp.p")
    print('Size (MB):', os.path.getsize("temp.p")/1e6)
    os.remove('temp.p')
print_size_of_model(model)
```

**Comment by AIbeginner2020** [comment]
By using the print_size_of_model method, the pruned model size shown did'nt reduce because I think this is cause by even the value is prune （turn to 0），but is still store with fp32. So I wonder how did you get your pruned model size? 
  
![image](https://user-images.githubusercontent.com/71377872/119232212-f233e180-bb56-11eb-9842-e5519d1cfbb0.png)

**Comment by Aarsh2001** [comment]
Yeah so you're using Pytorch's pruning which is unstructured . what unstructured pruning does is that it will put zeros in your weight matrix but it won't remove them. In essence you have a sparse model but you're still saving the zeros. If you want model size reduction use this library(torch_pruning), which uses structured pruning
